acl/acl-lib.pl

```diff
@@ -1299,6 +1299,19 @@ sub can_edit_user
 return 0;
 }
 
+=head2 can_module_acl(&mod)
+
+Returns 1 if the given module has ability to manage its own ACLs.
+
+=cut
+sub can_module_acl
+{
+my ($mod) = @_;
+my $mdir = &module_root_directory($mod);
+return 1 if (-f "$mdir/acl_security.pl" || -f "$mdir/config.info");
```

Comment:
What has config.info got to do with ACLs?

Reply:
> What has config.info got to do with ACLs?

It determines whether the "Can edit module configuration?" option will be displayed. If neither `acl_security.pl` nor `config.info` exists, there is no need to make it a link, as the opened page would still be empty and appear broken.

Reply:
Ah I see this makes sense